sdk/template-dpg/Azure.ServiceTemplate.Template/src/Azure.ServiceTemplate.Template.csproj

```diff
@@ -5,18 +5,15 @@
     <Version>1.0.0-beta.1</Version>
     <PackageTags>Azure Template</PackageTags>
     <TargetFrameworks>$(RequiredTargetFrameworks)</TargetFrameworks>
-    <NoWarn>$(NoWarn);419</NoWarn>
+    <NoWarn>$(NoWarn);419;AZC0012;AZC0016</NoWarn>
```

Comment:
We should remove all <NoWarn> blocks ... we don't want partners suppressing warnings this way.

Reply:
Then we need to add suppress files for those. Otherwise the build may fail. Is Adding suppress files acceptable solution?